tests/Unit/CartTest.php

```diff
@@ -4,18 +4,52 @@
 
 class CartTest extends TestCase
 {
+    /**
+     * If item is added to the cart
+     *
+     * @covers Cart::getItems
+     */
     public function testAddItem()
     {
         $cart = new Cart();
-
         $cart->addItem('foo');
         $this->assertEquals(array('foo'), $cart->getItems());
         $this->assertNotEquals(array('bar'), $cart->getItems());
     }
-    public function testGetItems()
+
+    /**
+     * Remove an item from the cart
+     *
+     * @covers Cart::getItems
+     */
+    public function testRemoveItem()
     {
         $cart = new Cart();
+        $cart->removeItem('foo');
+        $this->assertEmpty('', $cart->getItems());
```

Comment:
Asserting that an empty string is empty will always pass.

I think you might be misunderstanding the arguments that assertion takes. Here are the docs for that: https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertEmpty

That assertion _could_ absolutely work for you. What does `assertEmpty` take for arguments? How should your code change to correctly use that assertion?